theforeman.org/pipelines/release/source/foreman.groovy

```diff
@@ -1,5 +1,5 @@
 pipeline {
-    agent { label 'fast' }
+    agent any
```

Comment:
should this be `none`, like `theforeman.org/pipelines/test/foreman.groovy`, or is `any` required for `post` to work properly?

(Reading it again, "Build and Archive Source" also has no agent defined)

Reply:
It was my guess that it was needed for the source build

Reply:
I think I'd prefer a `none` here and an explicit `any` in the source build, but not blocking :)

Reply:
I looked at it but wasn't sure you could set it for an individual stage after reading the docs. I can take another look

Reply:
https://github.com/theforeman/jenkins-jobs/blob/2b2129bada42ef4ce1d68881d2f007ccbf7cb737/theforeman.org/pipelines/release/source/foreman.groovy#L77-L78

we do that today? :)

Reply:
Done. I also moved the `cleanup` to the specific stage because I think that will work better. I don't think the general notification steps in `post` need an agent.